docs/Config_Reference.md

```diff
@@ -2484,6 +2484,13 @@ pin:
 #shutdown_value:
 #   The value to set the pin to on an MCU shutdown event. The default
 #   is 0 (for low voltage).
+#host_acknowledge_timeout:
```

Comment:
Would something like `maximum_mcu_duration` be better?  It's not the host that has the timeout, but the mcu.